public/app/index.ts

```diff
@@ -19,4 +19,16 @@ if (window.nonce) {
 window.__grafana_app_bundle_loaded = true;
 
 import app from './app';
-app.init();
+
+const prepareInit = async () => {
+  if (process.env.frontend_dev_mock_api) {
+    return import('test/mock-api/worker').then((workerModule) => {
+      workerModule.default.start({ onUnhandledRequest: 'bypass' });
+    });
+  }
```

Comment:
Conditionally importing the worker so we don't include it in the bundle when we don't need to (could be quite large otherwise)